setup.py

```diff
@@ -10,19 +10,18 @@
     "click>=7.0",
     "rasterio>=1.1",
     "numpy~=1.15",
-    "morecantile>=3.0,<3.1",
+    "morecantile>=3.1,<4.0",
     "pydantic",
 ]
 
 extra_reqs = {
-    "test": ["pytest", "pytest-cov", "rio-tiler>=3.0.0a0", "cogdumper"],
-    "dev": ["pytest", "pytest-cov", "rio-tiler>=3.0.0a0", "pre-commit", "cogdumper"],
+    "test": ["pytest", "pytest-cov", "cogdumper"],
```

Comment:
removing rio-tiler to avoid circular dependency (rio-cogeo is a dependency of rio-tiler test suite)